packages/react-components/react-datepicker-compat/src/index.ts

```diff
@@ -1,41 +1,11 @@
-export { AnimationDirection, Calendar, calendarClassNames, useCalendarStyles_unstable } from './Calendar';
-export type { CalendarNavigationIcons, CalendarProps, CalendarStyleProps, CalendarStyles, ICalendar } from './Calendar';
+export { AnimationDirection, calendarClassNames } from './Calendar';
+export type { CalendarProps, ICalendar } from './Calendar';
 
-export { CalendarDay, calendarDayClassNames, useCalendarDayStyles_unstable } from './CalendarDay';
-export type { CalendarDayProps, CalendarDayStyleProps, CalendarDayStyles, ICalendarDay } from './CalendarDay';
+export { calendarDayClassNames } from './CalendarDay';
+export type { CalendarDayProps, ICalendarDay } from './CalendarDay';
```

Comment:
Since Calendar passes props down to CalendarMonth and CalendarDay, these should be exported for typing purposes.

Reply:
Do you need to add the `@internal` notation to these types?

Reply:
good question, I'll ask v-build!